site/content/en/docs/faq/_index.md

```diff
@@ -159,3 +159,20 @@ $env:MINIKUBE_HOME = "D:\.minikube"
 
 minikube start
 ```
+
+## I am in China or Iran, and I encounter errors when trying to start minikube, what should I do?
+
+After executing `minikube start`, minikube will try to pulling images from `gcr.io` or dockerhub. However, it has been confirmed that Chinese(mainland) and Iranian users may not have access to `gcr.io` or dockerhub. So in China mainland and Iran, it is very likely that `minikube start` will fail.
+
+For Chinese users, the reason is that China mainland government has set up GFW firewall to block any access to `gcr.io` or dockerhub from China mainland. For Iranian users, the reason is that `gcr.io` and dockerhub have banned all Iranian IPs due to sanctions.
+
+Minikube is an open community and we are always willing to help users from any corner of the world to use our open-source softwares, and provide possible assistance when possible. Here are 3 possible ways to get rid of the blockade.
+
+1. Use `minikube start --image-mirror-country='cn'` instead. Aliyun (a Chinese corporation) provides a mirror repository (`registry.cn-hangzhou.aliyuncs.com/google_containers`) for those images, to which both Chinese users or Iranian users have access. By using `--image-mirror-country='cn'` flag, minikube will try to pull the image from Aliyun mirror site as first priority. <br/><br/> *Note: when a new image is published on gcr.io, it may take several days for the image to be synchronized to Aliyun mirror repo. However, miniikube will always try to pull the newest image by default, which will cause a failure of pulling image. Under this circumstance, you HAVE TO use `--kubernetes-version` flag AS WELL to tell minikube to use an older version image which is available on Aliyun repo.* <br/><br/> *For example, `minikube start --image-mirror-country='cn'  --kubernetes-version=v1.23.8` will tell minikube to pull v1.23.8 k8s image from Aliyun.*
```

Comment:
```suggestion
1. Use `minikube start --image-mirror-country='cn'` instead. Aliyun (a Chinese corporation) provides a mirror repository (`registry.cn-hangzhou.aliyuncs.com/google_containers`) for those images, to which both Chinese users or Iranian users have access. By using `--image-mirror-country='cn'` flag, minikube will try to pull the image from Aliyun mirror site as first priority. <br/><br/> *Note: when a new image is published on gcr.io, it may take several days for the image to be synchronized to Aliyun mirror repo. However, minikube will always try to pull the newest image by default, which will cause a failure of pulling image. Under this circumstance, you HAVE TO use `--kubernetes-version` flag AS WELL to tell minikube to use an older version image which is available on Aliyun repo.* <br/><br/> *For example, `minikube start --image-mirror-country='cn'  --kubernetes-version=v1.23.8` will tell minikube to pull v1.23.8 k8s image from Aliyun.*
```